src/redux/reducer.js

```diff
@@ -53,10 +53,16 @@ export const reducer = createReducer(initialState, {
 
     [OPEN_STUDY]: (state, action) => {
         state.studyName = action.studyName;
+        state.description = action.description;
```

Comment:
You don't need these fields when the study is open?

Reply:
Done